.env.example

```diff
@@ -1,2 +0,0 @@
-PORT=8080
```

Comment:
Why did you remove this file?

Reply:
I just moved it to [.env](https://github.com/botim/backend/blob/869d1193bf8eb04790b74081a629dc608407f40b/.env) file. gitignore had `.env` so it didn't commit it so I committed it manually for reference for now but I think it's good to keep it in gitignore so we don't accidentally commit secrets